gravitee-am-ui/src/app/domain/settings/password-policies/domain-password-policies.component.html

```diff
@@ -0,0 +1,46 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+    
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    
+            http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div *ngIf="!isEmpty()" class="gv-page-container" fxFlex fxLayout="column" fxLayoutAlign="center center">
+  <app-emptystate
+    [message]="'Password policies will appear here'"
+    [subMessage]="'Start configuring your password policies here'"
+    [icon]="'donut_large'"
+  ></app-emptystate>
+  <button mat-raised-button color="primary" type="button" style="margin-top: 10px">Create Password Policy</button>
+</div>
+<div *ngIf="isEmpty()" class="gv-page-container">
```

Comment:
**issue**: I think this is the case where the isEmpty() return false. isn't it ?